packages/glob/src/internal-globber.ts

```diff
@@ -128,6 +128,11 @@ export class DefaultGlobber implements Globber {
         continue
       }
 
+      // Hidden file or directory?
```

Comment:
What upload-artifact version(s) is this code path exercised in?

Reply:
Glob is utilized here https://github.com/actions/upload-artifact/blob/main/src/shared/search.ts